src/corpus/application/TextService.ts

```diff
@@ -233,7 +237,7 @@ export default class TextService {
     ]).then(
       ([note, parallelLines, intertext]) =>
         new LineVariantDetails(
-          variant.reconstruction,
+          setSentenceIndices(variant.reconstruction),
```

Comment:
This seems to be the only place where `setSentenceIndices` needs to be called for the alignment to work.